electron-builder.yaml

```diff
@@ -16,9 +16,12 @@ mac:
   hardenedRuntime: true
   icon: buildResources/ScratchDesktop.icns
   provisioningProfile: embedded.provisionprofile
+  artifactName: "Scratch ${version}.${ext}"
```

Comment:
The `artifactName` is the name of the package file on disk. The DMG, for example, would default to `Scratch 3 3.15.0.dmg` without this.

Reply:
How does yaml know what ${version} and ${ext} are?

Reply:
These aren't filled in at the YAML level: this goes into `electron-builder`, which fills in the template during the build process. Here are the relevant sections of the `electron-builder` docs:
https://www.electron.build/configuration/configuration#artifact-file-name-template (somewhat light...)
https://www.electron.build/file-patterns#file-macros